src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -33,6 +34,38 @@ using namespace tvg;
 
 static const char* NoError = "None";
 
+#ifdef THORVG_WG_RASTER_SUPPORT
+    static WGPUInstance instance{};
+    static WGPUAdapter adapter{};
+    static WGPUDevice device{};
```

Comment:
@SergeyLebedkin 
Browser WebGPU will have bit different approach other than Native WebGPU.

WASM will ensure WGPUInstance, WGPUAdapter and WGPUDevice in singleton member. They will only be initialized once before used in player(s).

Reply:
You don't need to static them, please just adding a boolean (i.e: initialized) for avoiding the multiple initialization.